mm/2s2h/Rando/Logic/Regions/SpiderHouses.cpp

```diff
@@ -24,6 +24,7 @@ static RegisterShipInitFunc initFunc([]() {
             CHECK(RC_OCEAN_SPIDER_HOUSE_POT_MAIN_ROOM_2, true),
             CHECK(RC_OCEAN_SPIDER_HOUSE_POT_MAIN_ROOM_WEB, true),
             CHECK(RC_OCEAN_SPIDER_HOUSE_POT_MAIN_ROOM_BOE, true),
+            CHECK(RC_OCEAN_SPIDER_HOUSE_WALLET, true),
```

Comment:
Move this to RR_OCEAN_SPIDER_HOUSE_ENTRANCE_UPPER.

Reply:
not quite sure how the logic portion of the token access would go, seems like a lot but ill let @garrettjoecox make the final determination, i did move the wallet to the right spot, not sure why i put it there to begin with lol.